Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Diagnostics;
+
+namespace ConsoleApp2
+{
+    class Program
+    {
+        
+        static void Main(string[] args)
+        {
+            Segregator segregator = new Segregator();
```

Comment:
Test values needs to include empty set, set with single value, 2 value set with number repetition.